spring-cloud-dataflow-admin-local/src/main/resources/admin.yml

```diff
@@ -2,7 +2,11 @@ server:
   port: 9393
 management:
   contextPath: /management
-
+info:
```

Comment:
Currently we don't have a "base yaml" file in the admin-starter project. Also, wouldn't that mean that the properties are resolved against the `spring-cloud-dataflow-admin-starter`, probably not something we want to happen.